.github/pull_request_template.md

```diff
@@ -10,6 +10,7 @@ Thanks for submitting a pull request! Please provide enough information so that
 <!-- E.g. a bugfix, feature, refactoring, etc… -->
 
 **Issue Number:**
+<!-- Pick one of the below options -->
```

Comment:
```suggestion
<!-- Pick one of the below options.  Please remove those which don't apply. -->
```

I really want to call out that they should only include the items which make sense.  Less information for the PR reader to parse is better.